python/ql/src/Classes/CallsToInitDel/MethodCallOrder.qll

```diff
@@ -0,0 +1,213 @@
+/** Definitions for reasoning about multiple or missing calls to superclass methods. */
+
+import python
+import semmle.python.ApiGraphs
+import semmle.python.dataflow.new.internal.DataFlowDispatch
+import codeql.util.Option
+
+/** Holds if `meth` is a method named `name` that transitively calls `calledMulti` of the same name via the calls `call1` and `call2`. */
+predicate multipleCallsToSuperclassMethod(
+  Function meth, Function calledMulti, DataFlow::MethodCallNode call1,
+  DataFlow::MethodCallNode call2, string name
+) {
+  exists(Class cls |
+    meth.getName() = name and
+    meth.getScope() = cls and
+    call1.getLocation().toString() < call2.getLocation().toString() and
```

Comment:
Isn't using `toString` in queries discouraged?

Reply:
That is true.  
Probably best to compare the locations start and end lines directly.
Have updated.